docs/changelog/97962.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 97962
+summary: Remove synonyms feature flag and related classes
```

Comment:
Let's modify this to "Include release notes that describe the overall feature from a customer perspective."